xarray/coding/cftime_offsets.py

```diff
@@ -171,6 +174,38 @@ def _get_offset_day(self, other):
         return _get_day_of_month(other, self._day_option)
 
 
+class Tick(BaseCFTimeOffset):
+    # analogous https://github.com/pandas-dev/pandas/blob/ccb25ab1d24c4fb9691270706a59c8d319750870/pandas/_libs/tslibs/offsets.pyx#L806
+
+    def _next_higher_resolution(self):
+        self_type = type(self)
+        if self_type not in [Day, Hour, Minute, Second, Millisecond]:
+            raise ValueError("Could not convert to integer offset at any resolution")
+        if type(self) is Day:
+            return Hour(self.n * 24)
+        if type(self) is Hour:
+            return Minute(self.n * 60)
+        if type(self) is Minute:
+            return Second(self.n * 60)
+        if type(self) is Second:
+            return Millisecond(self.n * 1000)
+        if type(self) is Millisecond:
+            return Microsecond(self.n * 1000)
+
+    def __mul__(self, other):
```

Comment:
Similar to in `BaseCFTimeOffset` I think it would be good to return `NotImplemented` for any invalid `other`.
```suggestion
    def __mul__(self, other):
        if not isinstance(other, (int, float)):
            return NotImplemented
```